services/adgs/config/adgs_search_config.yaml

```diff
@@ -12,87 +12,117 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-collections:
-  - id: s1_adgs_OPER_AUX_ECMWFD_PDMC
-    station: adgs
-    query:
-      productType: OPER_AUX_ECMWFD_PDMC
-    stac_extensions: [ "https://stac-extensions.github.io/eo/v1.0.0/schema.json", "https://stac-extensions.github.io/projection/v1.0.0/schema.json", "https://stac-extensions.github.io/view/v1.0.0/schema.json" ]
-    title: "'OPER_AUX_ECMWFD_PDMC' products"
-    description: "'OPER_AUX_ECMWFD_PDMC' products"
-    license: other
-    extent:
-      spatial:
-        bbox: [[ -180, -82.85, 180, 82.82 ]]
-      temporal:
-        interval: [[ '2024-06-12T02:57:21.459000Z', '2024-08-22T11:30:12.767000Z' ]]
-    links:
-      - rel: license
-        href: 'https://scihub.copernicus.eu/twiki/pub/SciHubWebPortal/TermsConditions/Sentinel_Data_Terms_and_Conditions.pdf'
-        title: 'Legal notice on the use of Copernicus Sentinel Data and Service Information'
-    providers:
-      - name: 'European Union/ESA/Copernicus'
-        roles:
-          - producer
-          - licensor
-        url: 'https://sentiwiki.copernicus.eu/web/s1-mission'
-      - name: 'Reference System'
-        roles:
-          - host
-        url: 'https://home.rs-python.eu/'
-
-  - id: s2_adgs2_AUX_OBMEMC
-    station: adgs2
-    query:
-      productType: AUX_OBMEMC
-    stac_extensions: [ "https://stac-extensions.github.io/eo/v1.0.0/schema.json", "https://stac-extensions.github.io/projection/v1.0.0/schema.json", "https://stac-extensions.github.io/view/v1.0.0/schema.json" ]
-    title: "'AUX_OBMEMC' products"
-    description: "'AUX_OBMEMC' products"
-    license: other
-    extent:
-      spatial:
-        bbox: [[ -180, -82.85, 180, 82.82 ]]
-      temporal:
-        interval: [[ '2024-06-12T02:57:21.459000Z', '2024-08-22T11:30:12.767000Z' ]]
-    links:
-      - rel: license
-        href: 'https://scihub.copernicus.eu/twiki/pub/SciHubWebPortal/TermsConditions/Sentinel_Data_Terms_and_Conditions.pdf'
-        title: 'Legal notice on the use of Copernicus Sentinel Data and Service Information'
-    providers:
-      - name: 'European Union/ESA/Copernicus'
-        roles:
-          - producer
-          - licensor
-        url: 'https://sentiwiki.copernicus.eu/web/s1-mission'
-      - name: 'Reference System'
-        roles:
-          - host
-        url: 'https://home.rs-python.eu/'
+# THIS FILE WAS AUTOMATICALLY CREATED FROM:
```

Comment:
This file was automatically created from the template when running `poetry run python resources/sync_config_files.py`